Wordle.Api/Wordle.Api/Controllers/WordEditorController.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Dtos;
+using Wordle.Api.Identity;
+using Wordle.Api.Models;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WordEditorController(WordEditorService wordEditorService) : ControllerBase
+    {
+        [Authorize(Policy = Policies.CanAddDeleteWords)]
+        [HttpPost("DeleteWord")]
+        public async Task<IActionResult> DeleteWord(string word)
+        {
+            var result = await wordEditorService.DeleteWordAsync(word);
+
+            return result ? Ok() : BadRequest();
+        }
+        

```

Comment:
it seems you are returning true and false to tie in with the var result, not sure if this can be implemented in the front end for these requests instead of here

Reply:
Good catch, will work on that.